### PR TITLE
Gamanraid Pedigree

### PR DESCRIPTION
**Pull request by Eystein-Thanisch**
Precedes Clann Morna Tract in Lec and provides context for it.

### CODE REVIEW

File: Lec/gamanraid.trig [hunk 1]
```diff
@@ -0,0 +1,241 @@
+@base <http://example.com/LL/gamanraid.trig> .
```
Comment (cyocum) [inline Lec/gamanraid.trig]: Lec needed here not LL

File: Lec/gamanraid.trig [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    irishRel:nomName "Tuam Tened";
+    rel:childOf <#FhirDaBenn>;
+    owl:sameAs <http://example.com/Lec/clann_morna.trig#TuamaTened>.
+
+<#FhirDaBenn>
+    a foaf:Person;
+    irishRel:nomName "Fer Da Benn";
+    irishRel:genName "Fhir Da Benn";
+    rel:childOf <#DoilbRindGlais>;
+    rdfs:comment ".i. bend chuic oigli agus benn boirnne uair ba ri seom eturru sin";
```
Comment (cyocum) [inline Lec/gamanraid.trig]: @sga at the end of the string?

File: Lec/gamanraid.trig [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+    irishRel:genName "Fhaelchon Fhuilig";
+    irishRel:nomName "Faelchú Fuilech";
+    rel:childOf <#FhirDaThonn>.
+
+<#FhirDaThonn>
+    a foaf:Person;
+    irishRel:genName "Fhir Da Thonn";
+    irishRel:nomName "Fer Da Thonn";
+    rel:childOf <#FhirDumaMoir>;
+    rdfs:comment ".i. tond indsi caerach i corco baiscind agus tonn indsi gluair i tuaid".
```
Comment (cyocum) [inline Lec/gamanraid.trig]: "@sga" at the end of the string?

**Review by cyocum — CHANGES_REQUESTED** [review]
Cool.  Just a couple of things and we should be ready to go.

**Comment by Eystein-Thanisch** [comment]
Thanks. Done.

**Review by cyocum — APPROVED** [review]

**Pull request merged**